crates/cheatnet/src/cheatcodes/l1_handler_execute.rs

```diff
@@ -29,68 +29,70 @@ impl CheatnetState {
         paid_fee_on_l1: &Felt252,
         payload: &[Felt252],
     ) -> Result<(), CheatcodeError> {
-        let blockifier_state: &mut CachedState<ExtendedStateReader> = &mut self.blockifier_state;
+        anyhow!("l1_handler_execute is temporarily not supported in this version, use a different version of starknet foundry");
+        Ok(())
+        // let blockifier_state: &mut CachedState<ExtendedStateReader> = &mut self.blockifier_state;
```

Comment:
We should leave a TODO

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/753/commits/d893e4b6cbaa75ffdcf690a55374b0fb362b52b5